src/function/vector_string_operations.cpp

```diff
@@ -141,5 +135,71 @@ std::vector<std::unique_ptr<VectorOperationDefinition>> SubStrVectorOperation::g
     return definitions;
 }
 
+std::vector<std::unique_ptr<VectorOperationDefinition>>
+RegexpFullMatchVectorOperation::getDefinitions() {
+    std::vector<std::unique_ptr<VectorOperationDefinition>> definitions;
+    definitions.emplace_back(make_unique<VectorOperationDefinition>(REGEXP_FULL_MATCH_FUNC_NAME,
+        std::vector<DataTypeID>{STRING, STRING}, BOOL,
+        BinaryExecFunction<ku_string_t, ku_string_t, uint8_t, operation::RegexpFullMatch>,
+        BinarySelectFunction<ku_string_t, ku_string_t, operation::RegexpFullMatch>,
+        false /* isVarLength */));
+    return definitions;
+}
+
+std::vector<std::unique_ptr<VectorOperationDefinition>> RegexpMatchesOperation::getDefinitions() {
+    std::vector<std::unique_ptr<VectorOperationDefinition>> definitions;
+    definitions.emplace_back(make_unique<VectorOperationDefinition>(REGEXP_MATCHES_FUNC_NAME,
+        std::vector<DataTypeID>{STRING, STRING}, BOOL,
+        BinaryExecFunction<ku_string_t, ku_string_t, uint8_t, operation::RegexpMatches>,
+        BinarySelectFunction<ku_string_t, ku_string_t, operation::RegexpMatches>,
+        false /* isVarLength */));
+    return definitions;
+}
+
+std::vector<std::unique_ptr<VectorOperationDefinition>> RegexpReplaceOperation::getDefinitions() {
```

Comment:
Add a TODO here saying that we need to support regexReplace with modifier in the future.